libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Preds/LedgerState.hs

```diff
@@ -130,11 +130,14 @@ demo mode = do
           >>= (\subst -> monadTyped $ substToEnv subst emptyEnv)
       )
   lstate <- monadTyped $ runTarget env (ledgerStateT proof)
+  let _env2 = getTarget lstate (ledgerStateT proof) emptyEnv
```

Comment:
Dead code?
```suggestion
```

Reply:
Actually we are testing env2, but while tracking down some bug, I switched to the earlier 'env', it is back where it should be. Good catch.